PROTOCOL.md

```diff
@@ -424,6 +429,66 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests can be sent with `|RQID`. `RQID` is a unique number used to
+identify which action the request was intended for and is used to protect
+against race conditions with the cancel button.
+
+If an invalid request is sent, the game will replace the missing or
+erroneous request with the first valid choice, which is usually the first
```

Comment:
We also no longer need "or Struggle if there are no usable moves left", since "usually" already covers that there are exceptions.